impl/src/test/java/org/ehcache/impl/internal/store/tiering/CacheStoreTest.java

```diff
@@ -680,6 +683,36 @@ public void testReleaseStoreFlushes () throws CacheAccessException {
     verify(cachingTierProvider, times(1)).releaseCachingTier(any(CachingTier.class));
   }
 
+  @Test
+  public void testRank() throws Exception {
+    CacheStore.Provider provider = new CacheStore.Provider();
+
+    Assert.assertThat(provider.rank(
+        Collections.<ResourceType>singleton(ResourceType.Core.DISK), Collections.<ServiceConfiguration<?>>emptyList()),
+        Matchers.is(0));
```

Comment:
Most existing references to `ResourceType.Core` aren't statically imported.  I've introduced a method that cleans this up considerably -- have a look at the newer stuff when it gets pushed and let me know if you still think a static import will help.